src/test/java/seedu/duke/command/EditCommandTest.java

```diff
@@ -18,12 +23,12 @@
     @BeforeEach
     public void setUp() throws Exception {
         testList = new ItemContainer("test");
-        testList.addItem(new Item("HarryPotter", "16.1", "25.12"));
-        testCommand = new EditCommand("HarryPotter", "16.1", "20", testList);
     }
 
     @Test
-    public void execute_oneItemAlreadyInList_editsNormally() {
+    public void execute_oneItemAlreadyInList_editsNormally() throws Exception {
+        testList.addItem(new Item("HarryPotter", "16.1", "25.12"));
+        testCommand = new EditCommand("HarryPotter", "16.1", "20");
```

Comment:
the throw exception here, is it because the editCommand, addItem etc throws a lot of different exceptions?